### DISCUSSION TITLE
enebularでのネガポジAPI～LINEAPIのつなぎ方

### DISCUSSION THREAD

**Discussion by slc-rd** [1ft-seabass/ma-hackathon-qa-and-knowledge-202210#21] [Q&A] [answered]
■目指してるところ
enebularでLINEから受け取ったメッセージを、ネガポジAPIで判定させて判定結果をもとに
画像でLINEに返信を返したい。

■状況
enebularでネガポジAPI～LINEAPIをつなげると、LINEへの返信がうまくつなげられない。
msg.payloadに格納されている情報の理解が混乱しています…。

■できてること
・LINE→ネガポジAPIの判定
・LINE→LINEのメッセージ返信

できればチーム（f_ええんやで）まで来ていただいて、画面共有しながらご相談したいです。

**Comment by 1ft-seabass** [comment, marked as answer]
確認しまーす

**Reply by 1ft-seabass** [reply]
対応したメモ

- ネガポジ API から LINE API に渡す前に msg.headers のデータを削除してきれいにしておく

**Reply by 1ft-seabass** [reply]
@slc-rd これでうまく対応できたっぽいですね！

**Reply by slc-rd** [reply]
ありがとうございました！
目標の第一段階は突破できそうです。
